test/functional/zmq_test.py

```diff
@@ -63,25 +69,49 @@ def _zmq_test(self):
         body = msg[1]
         msgSequence = struct.unpack('<I', msg[-1])[-1]
         assert_equal(msgSequence, 0)  # must be sequence 0 on hashtx
+        txhash = body
```

Comment:
why not just assign `txhash` on line 69. There's no reason to have `body` in this section.